### ISSUE TITLE
Theme release: can we update screenshot.png?

### ISSUE THREAD

**Issue by ericfoy** [backdrop-ops/backdropcms.org#989]
It would be nice if backdropcms.org would re-upload a revised "screenshot.png" file when said file changes in a new theme release. As it is, the original image (which, in my case, got vertically cropped and just looks bad) remains...

**Comment by jenlampton** [comment]
Yes, this is a good idea. I think the change would need to be made to https://github.com/backdrop-ops/backdropcms.org/blob/main/www/modules/custom/borg_github/borg_github.module?

**Comment by ericfoy** [comment]
Thanks, @jenlampton, I'll look into that.

**Comment by jenlampton** [comment]
@ericfoy also -- and sorry, I should have done this first -- I have made you the owner of the project node on b.org, which means you should be able to edit it now: https://backdropcms.org/project/modoc. This will enable you to manually replace the screenshot that shows on b.org and in the project browser until we can get the code problem sorted out.

**Comment by ericfoy** [comment]
@jenlampton : Oh, wow. Awesome. I never even thought about that possibility... 

Also, don't worry, I understood your original comment exactly how you meant it. I also found it rather empowering, like... "Hey, it's open source. If I don't like the way it works, I can fix it myself!" Plus, I have an entire dev team available to keep me from screwing it up!
